.github/workflows/main.yml

```diff
@@ -106,12 +118,14 @@ jobs:
             artifact: "LinuxA.tar.xz"
             os: ubuntu-latest
             build_type: "debug"
-            cpp: disable
+            cpp: enable
             fortran: disable
             java: disable
             ts: disable
             hl: enable
             parallel: disable
+            mirror_vfd: disable
+            direct_vfd: enable
```

Comment:
The mirror VFD still spits out warnings, so it can't be enabled here until I fix that.